### PR TITLE
Updates Kotha et al (2019 - Now 2020) SERA GMPE

### PR DESCRIPTION
**Pull request by g-weatherill**
Following another (hopefully final) round of regressions the coefficients for the Kotha et al. (2019) GMPE have been updated. These are now implemented and all coefficients tables, test tables and downstream tests (e.g. AvgSA gmpe, classical test case) are updated. 

As the GMPE itself is now under review in a journal the publication date will be 2020 (again, hopefully), so the `KothaEtAl2019` names are changed to 2020, but as the GMPE still retains the experimental warning then no attempt is made to ensure backward compatibility. Current users of the model for the SERA project (to my knowledge) will be informed of the change by me personally.

### CODE REVIEW

**Comment by micheles** [comment]
The tests are running here: https://ci.openquake.org/job/zdevel_oq-engine/2865/

Please solve the conflict (it looks like and indentation conflict).

File: openquake/hazardlib/gsim/kotha_2020.py [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+            self.c3 = CoeffsTable(sa_damping=5, table=c3in)
+        else:
+            self.c3 = c3
+
+        self.sigma_mu_epsilon = sigma_mu_epsilon
+        self.ergodic = ergodic
+        if self.sigma_mu_epsilon:
+            # Connect to hdf5 and load tables into memory
+            self.retreive_sigma_mu_data()
+        else:
```
Comment (micheles) [inline openquake/hazardlib/gsim/kotha_2020.py]: Is this an US/British spelling issue, or a bona fide misprint? retreive -> retrieve

Comment (g-weatherill) [reply openquake/hazardlib/gsim/kotha_2020.py]: It's a typo. Now fixed

File: openquake/hazardlib/gsim/kotha_2020.py [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+        ampl[sites.vs30measured] = (C["d0_obs"] + C["d1_obs"] *
+                                    np.log(vs30[sites.vs30measured]))
+        # For inferred Vs30 sites
+        idx = np.logical_not(sites.vs30measured)
+        ampl[idx] = (C["d0_inf"] + C["d1_inf"] * np.log(vs30[idx]))
+        return ampl
+
+    def get_stddevs(self, C, stddev_shape, stddev_types, sites):
+        """
+        Returns the standard deviations, with different site standard
```
Comment (mmpagani) [inline openquake/hazardlib/gsim/kotha_2020.py]: This note is a bit cryptic

File: openquake/hazardlib/tests/gsim/kotha_2020_test.py [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>
+from openquake.hazardlib.gsim.base import CoeffsTable
+from openquake.hazardlib.gsim.kotha_2020 import (KothaEtAl2020SERA,
+                                                 KothaEtAl2020Site,
+                                                 KothaEtAl2020Slope,
+                                                 KothaEtAl2020SERASlopeGeology)
+from openquake.hazardlib.tests.gsim.utils import BaseGSIMTestCase
+
```
Comment (mmpagani) [inline openquake/hazardlib/tests/gsim/kotha_2020_test.py]: Would be useful to have a comment explaining how you computed the verification tables

**Pull request merged**